source/index-management/tests/createSearchIndex.yml

```diff
@@ -0,0 +1,62 @@
+description: "createSearchIndex convenience helper"
+schemaVersion: "1.10"
```

Comment:
What about these test files require 1.10? That was when the `thread` entity was introduced (see: [changelog](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#changelog)).

Per [schemaVersion](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#schemaversion):

> Test files SHOULD be conservative when specifying a schema version. For example, if the latest schema version is 1.1 but the test file complies with schema version 1.0, the test file should specify 1.0.

This plays nicely with CI, which uses the test file's `schemaVersion` field for validation.